units/en/unit5/run-locally.mdx

```diff
@@ -0,0 +1,33 @@
+# Run locally
+
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://githubtocolab.com/dylanebert/ml-for-3d-course-notebooks/blob/main/capstone.ipynb)
+
+The instructions below are tested on an RTX 4090 on WSL2 Ubuntu 22.04. Instructions will differ and may not work, depending on your setup.
+
+1. Install `git`, `python 3.10`, and `cuda` if not already installed.
```

Comment:
Curious q: does this work locally on mac?

Note: people can also run the space via docker locally: https://huggingface.co/docs/hub/en/spaces-run-with-docker

Would be good to add that as an example too (it's okay to skip it if you think it's too much complexity)